README.md

```diff
@@ -111,7 +116,8 @@ and Windows.
 KerasCV provides access to pre-trained models via the `keras_cv.models` API.
 These pre-trained models are provided on an "as is" basis, without warranties
 or conditions of any kind.
-The following underlying models are provided by third parties, and subject to separate licenses:
+The following underlying models are provided by third parties, and subject to separate
```

Comment:
We have a disclaimer section for KerasNLP we actually got a few eyes on to vet the language for. Maybe we can just copy that here?

https://github.com/keras-team/keras-nlp#disclaimer